src/components/Header/Header.js

```diff
@@ -0,0 +1,605 @@
+			setLoginButton('로그인');
+		}
+	};
+	const closeLoginModal = () => {
+		setLoginModalVisible(false);
+	};
+
+	const [searchModalVisible, setSearchModalVisible] = useState(false);
+	const openSearchModal = () => {
+		setSearchModalVisible(true);
+	};
+	const closeSearchModal = () => {
+		setSearchModalVisible(false);
+	};
+
+	const [searchLocation, setSearchLocation] = useState('');
+	const handleInput = e => {
+		setSearchLocation(e.target.value);
+	};
+
+	const [startDate, setStartDate] = useState(new Date());
+	const [endDate, setEndDate] = useState(null);
+	const onChange = dates => {
+		const [start, end] = dates;
+		setStartDate(start);
+		setEndDate(end);
+	};
+
+	const checkInDate = `${startDate.getFullYear()}-${
+		startDate.getMonth() + 1
+	}-${startDate.getDate()}`;
+	const checkOutDate = endDate
+		? `${endDate.getFullYear()}-${endDate.getMonth() + 1}-${endDate.getDate()}`
+		: null;
+
+	const searchLinkAddress =
+		checkInDate && checkOutDate
+			? `/${searchLocation}/room?checkin=${checkInDate}&checkout=${checkOutDate}`
+			: `/${searchLocation}/room`;
+
+	const [isOpen, setIsOpen] = useState(false);
+
+	const handleClick = e => {
+		e.preventDefault();
+		setIsOpen(!isOpen);
+	};
+
+	return (
+		<HeaderWrapper>
+			<Link to="/">
+				<Logo>
+					{isMain ? (
+						<LogoImage src="../../images/logo1.png" />
+					) : (
+						<LogoImage src="../../images/logo2.png" />
+					)}
+
+					<LogoName>ourbnb</LogoName>
+				</Logo>
+			</Link>
+			<HeaderButtons>
+				<SelectedHeaderButton>
+					<HeaderLabel>
+						<HeaderInput>숙소</HeaderInput>
+					</HeaderLabel>
+				</SelectedHeaderButton>
+				<HeaderButton>
+					<HeaderLabel>
+						<HeaderInput>체험</HeaderInput>
+					</HeaderLabel>
+				</HeaderButton>
+				<HeaderButton>
+					<HeaderLabel>
+						<HeaderInput>온라인 체험</HeaderInput>
+					</HeaderLabel>
+				</HeaderButton>
+			</HeaderButtons>
+			<HeaderOption>
+				<HeaderChoice>호스트 되기</HeaderChoice>
+				<HeaderChoice>
+					<FiGlobe />
+				</HeaderChoice>
+			</HeaderOption>
+			<Profile onClick={openProfileModal}>
+				<Lines>
+					<Line />
+					<Line />
+					<Line />
+				</Lines>
+				<User>
+					<IoPersonCircle />
+				</User>
+			</Profile>
+			{profileModalVisible && (
+				<ProfileModal
+					visible={profileModalVisible}
+					closable={true}
+					maskClosable={true}
+					onClose={closeProfileModal}
+				>
+					<ModalListTop>
+						<Signup>회원가입</Signup>
+					</ModalListTop>
+					<ModalList onClick={openLoginModal}>
+						<Login>{loginButton}</Login>
+					</ModalList>
+					{loginModalVisible && (
+						<LoginModal
+							visible={loginModalVisible}
+							closable={true}
+							maskClosable={true}
+							onClose={closeLoginModal}
+						>
+							<LoginLogoImage src="../../images/logo2.png" />
+							<LoginLogo>아워비앤비 회원가입 및 로그인</LoginLogo>
+							<ModalLine />
+							<LoginComment>아워비앤비에 오신 것을 환영합니다.</LoginComment>
+							<KakaoLoginButton>
+								<LoginButton />
+							</KakaoLoginButton>
```

Comment:
```suggestion
							<KakaoLoginButton>
								<LoginLink />
							</KakaoLoginButton>
```

이미 클리커블한 버튼에 속해있어서 링크라던지 무언가 다른 이름을 주셔도 좋을거같습니다~